pkg/metrics/sync.go

```diff
@@ -134,6 +147,48 @@ func Sync(ctx context.Context, tmc *TendermintClient, st *Store) (uint, error) {
 	}
 }
 
+type Message struct {
+	Path    string `json:"path"`
+	Details string `json:"details"`
+}
+
+func messageDetails(msg weave.Msg) (string, error) {
```

Comment:
Another special case are gov proposals where the content is serialized. You should deserialze it as well to make it useful. There is an example in bnscli view

Reply:
This was not really part of the reqs for this ticket, so I left it out.

Reply:
Understood. Requirements are not fully clear to me. Let's have this in another PR if required // cc @davepuchyr

Reply:
As is, we have what we need for our submission to Ledger.  If we need more then @annatorok will add to the backlog.